examples/seismic/acoustic/acoustic_example.py

```diff
@@ -52,25 +52,25 @@ def acoustic_setup(dimensions=(50, 50, 50), spacing=(15.0, 15.0, 15.0),
 
 def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
         time_order=2, space_order=4, nbpml=40, full_run=False, **kwargs):
-
+    autotune = kwargs.pop('autotune', False)
```

Comment:
move ``autotune=False`` to the function signature so you don't have to pop it here. Same in TTI